File: All_File/Coding_Practice/ArrayCompitive/removeDuplicate.c
#include<stdio.h>
int main(){

int a[6]={2,7,9,2,3,4};
int i,j;
for(i=0;i<6;i++){
	for(j=i+1;j<6;j++){
		if(a[j]==a[i]){
			a[i]=-1;
			    temp=a[i];
		}
	
	}
		printf("%d",a[i]);
}

}

File: All_File/DSa/DSA/ARRAY/RemoveDuplicate.java
public class RemoveDuplicate {
    public static void main(String[] args) {
        int[] arr = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 7, 1, 6, 1, 8, 9 };
        int len = arr.length;
        
        int temp = 0;
        for (int i = 1; i < arr.length; i++) {
            for (int j = 0; j < arr.length - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    temp = arr[j + 1];
                    arr[j + 1] = arr[j];
                    arr[j] = temp;
                }
            }
        }
        int j = 0;
        for (int i = 0; i < len-1; i++) {
            if (arr[i] != arr[i + 1])
                arr[j++] = arr[i];
        }
        arr[j++] = arr[len - 1];
        for(int i=0;i<j;i++)
System.out.println(arr[i]);
    }
}
